packages/components/src/range-control/index.tsx

```diff
@@ -138,7 +138,8 @@ function UnforwardedRangeControl< IconProps = unknown >(
 		onChange( nextValue );
 	};
 
-	const handleOnChange = ( next: string ) => {
+	const handleOnChange = ( next?: string ) => {
+		// @ts-expect-error TODO: Investigate if this is problematic
```

Comment:
Should this use `ensureNumber` ?

Reply:
This one can't be fixed by `ensureNumber` because the problem is that `next` can potentially be undefined.

Technically, we can remove the ts-ignore without changing effective runtime behavior by doing something like this:

```jsx
let nextValue = typeof next === 'undefined' ? NaN : parseFloat( next );
```

But then again, I really think this one needs further investigation because it's unclear whether it's intended for `setValue()` to sometimes receive a NaN. It could actually be problematic, so I wouldn't want to imply that this behavior is legit.